### PR TITLE
added etherscan link to profile page

### PR DESCRIPTION
**Pull request by Royal-lobster**
Closes https://github.com/EveripediaNetwork/issues/issues/603

# Links
- (without other social links) https://monopedia-ui-git-profile-link-prediqt.vercel.app/account/kesar.eth
- (with existing social links) https://monopedia-ui-git-profile-link-prediqt.vercel.app/account/srujangurram.eth

# Screenshots
CleanShot 2022-08-02 at 19 30 44@2x

### CODE REVIEW

File: src/components/Layout/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -68,6 +68,8 @@ const Navbar = () => {
 
   const handleChainChanged = useCallback(
     (chainDetails: string) => {
+      console.log('chainDetails', chainDetails)
+      console.log('chainId', chainId)
```
Comment (Royal-lobster) [inline src/components/Layout/Navbar/Navbar.tsx:72]:
```suggestion
```

**Review by Simplypheyie96 — APPROVED** [review]

**Comment by kesar** [comment]
i think link is broken in prod.

also i would suggest a real etherscan icon:( image

not sure if people will understand the current one

**Comment by Simplypheyie96** [comment]
@Royal-lobster 
done

image

**Comment by Royal-lobster** [comment]
> @Royal-lobster 
> done
> 
> image
> 

Thanks @Simplypheyie96 ! i will make the changes.

**Pull request merged**